### ISSUE TITLE
Transition Z-Offset Buttons to Sliders

### ISSUE THREAD

**Issue by acikek** [ModFest/glowcase#44]
[Reference implementation](https://github.com/ModFest/glowcase/blob/rewrite-2.0/src/main/java/net/modfest/glowcase/client/gui/screen/ingame/TextBlockEditScreenV2.java#L44)

![image](https://github.com/user-attachments/assets/8fb81fa7-39a3-4a76-99ad-8360e4e4cb38)

**Comment by sleepingdragoninn** [comment]
To simplify this i would:
- Show only the slider and have its options only be the preset values by default
- When holding shift, swap to a free -0.4 to 0.4 slider
- When value is not set to a preset value, display text box on the right

Or:
- Have a preset slider
- Have a input box to the right of it (the only way to manually adjust offset)

**Comment by acikek** [comment]
I agree on the shift-to-free slider (opposite of what I've proposed in the screenshot) but I think the textbox should be visible and editable at all times.

**Comment by sleepingdragoninn** [comment]
closing this because blocks that need it now have decimal offsets not an enum
